animations/lib/src/basics/08_fade_transition.dart

```diff
@@ -0,0 +1,83 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+class FadeTransitionDemo extends StatefulWidget {
```

Comment:
Consider adding a comment linking to the [AnimatedWidget docs](https://api.flutter.dev/flutter/widgets/AnimatedWidget-class.html) for examples of other common animated widgets.